src/renderer/pages/PlayerPage.js

```diff
@@ -109,7 +109,7 @@ function renderMedia (state) {
   var MediaTagName = state.playing.type
   var mediaTag = (
     <MediaTagName
-      src={state.server.localURL}
+      src={state.server.localURL + '/' + state.playing.fileIndex}
```

Comment:
Exactly like you say. Refer to [this line](https://github.com/feross/webtorrent-desktop/pull/871/files#diff-1f9e4aabf46e7be3ad02d963e0a2fc07L317).